### DISCUSSION TITLE
v0.22 Native AI object detection through new motion sensor entities

### DISCUSSION THREAD

**Discussion by Teevolution** [fwestenberg/reolink_dev#340] [Q&A]
Can someone confirm what this means? Does this mean HA can do some sort of onboard motion processing itself? 

I have Reolink cameras that have onboard AI motion processing and some that do not. All cameras bring in the entities for motion, person, pet and vehicle but only the ones with these functions onboard actually work. Is this the case that the integration cannot tell what functions the camera have? Or that motion detection can be done off camera?

Thanks for any pointers

**Comment by cpainchaud** [comment]
this integration currently creates all sensors regardless of the camera capabilities
